HunterPie.Core/Game/Rise/MHRGame.cs

```diff
@@ -184,7 +186,13 @@ private void GetDeathCounter()
             AddressMap.Get<int[]>("QUEST_DEATH_COUNTER_OFFSETS")
         );
 
+        int maxDeathsCounter = _process.Memory.Deref<int>(
+            AddressMap.GetAbsolute("QUEST_ADDRESS"),
+            AddressMap.Get<int[]>("QUEST_MAX_DEATHS_OFFSETS")
+        );
+
         Deaths = deathCounter;
+        MaxDeaths = maxDeathsCounter;
```

Comment:
```suggestion
        MaxDeaths = maxDeathsCounter;
        Deaths = deathCounter;
```

Invert the order so `Deaths` will dispatch the event when `MaxDeaths` has an updated value

Reply:
I followed your guidance, MaxDeaths didn't get value.

Reply:
Reworked to dispatch.
MaxDeaths failed to get a value, if not dispatched.